index.js

```diff
@@ -79,12 +80,18 @@ function attachmentFromComponents(components, query) {
                   attachment.url = marketComponent.url;
                   attachment.thumbnail.url = `https://nexus-stats.com/img/items/${encodeURIComponent(nexusComponent.title)}-min.png`;
                   attachment.description = `Query results for: "${query}"`;
+
+                  let nexusMedian = component.prices.median ? component.prices.median + 'p' : 'No data';
```

Comment:
these should probably be `const` instead of `let` since they're never modified. might make a small method to check it since we do it 4 times, like `safeValue(value)`

```javascript
saveValue(value) {
   return value ? `${value}p` : 'No data';
}
```

I know it probably seems unnecessary, but it'll keep the check consistent for the future.